pypureclient/flasharray/FA_2_10/models/smtp.py

```diff
@@ -21,7 +21,7 @@
 if typing.TYPE_CHECKING:
     from pypureclient.flasharray.FA_2_10 import models
 
-class SMTP(object):
+class Smtp(object):
```

Comment:
FYI: These classes are available to end-users, so renaming them might cause some issues.